ci-scripts/copy_to_archive

```diff
@@ -33,42 +32,53 @@ fi
 
 # Copy the logs
 if [ -d ${LOG_DIR}/isafw-logs ]; then
-	echo "Archiving logs to $ARCHIVE_DIR"
-	mv ${LOG_DIR}/isafw-report_*/* $ARCHIVE_DIR
+    echo "Archiving logs to $ARCHIVE_DIR"
+    mv ${LOG_DIR}/isafw-report_*/* $ARCHIVE_DIR
+fi
+
+if [ -d ${LOG_DIR}/cooker ]; then
+    for f in ${LOG_DIR}/cooker/**/console-latest.log; do
+      echo "Archiving console-latest.log file to $ARCHIVE_DIR"
+      mv "$f" $ARCHIVE_DIR
+    done
+fi
+
+# Copy license manifest file
+if [ -f ${LICENSE_DIR}/${IMAGE_BASENAME}-*20*/license.manifest ]; then
+    echo "Archiving license.manifest file to $ARCHIVE_DIR"
+    mv ${LICENSE_DIR}/${IMAGE_BASENAME}-*20*/license.manifest $ARCHIVE_DIR
 fi
 
-# If there is an Intel directory
-if [ -d ${IMAGE_DIR}/${INTEL_ARCH} ]; then
-    echo "Archiving Intel image to $ARCHIVE_DIR"
-    mv ${IMAGE_DIR}/${INTEL_ARCH}/${IMAGE_BASENAME}-*rootfs.wic $ARCHIVE_DIR
-    mv ${IMAGE_DIR}/${INTEL_ARCH}/${IMAGE_BASENAME}-*rootfs.wic.bmap $ARCHIVE_DIR
-    mv ${IMAGE_DIR}/${INTEL_ARCH}/${IMAGE_BASENAME}-*rootfs.wic.bz2 $ARCHIVE_DIR
-    mv ${IMAGE_DIR}/${INTEL_ARCH}/${IMAGE_BASENAME}-*20*.swu $ARCHIVE_DIR
+# Copy if image files exist
+if [ -f ${IMAGE_DIR}/**/${IMAGE_BASENAME}-*rootfs.wic ]; then
```

Comment:
Instead of looping through all variants, would it be possible to pass ${MACHINE} as argument to this script? The artifacts would then reside in ${IMAGE_DIR}/${MACHINE}/.

Reply:
what is the purpose? It worked well before, looping through the variant dir if it's presented. It brings more logic to yocto.groovy script. For me there are two considerable options: initial one (check if variant dir presented and copy artifacts ) of current one

Reply:
In case the workspace is reused between builds, looping may be problematic since it may move artifacts from previous builds. This is my concern.

Consider to add a few comments in the header of the script which explains what it does and any side-effects, like e.g. that the artifacts are actually moved and not copied as the script name indicates.